Dockerfile

```diff
@@ -1,10 +1,23 @@
-FROM node:lts-slim
+FROM node:16.15.1-alpine as base
+
+RUN apk --update --no-cache add --virtual .builds-deps build-base python3
```

Comment:
Needed to install the dependencies, because some of the packages used here require gyp